cmd/hubagent/workload/setup.go

```diff
@@ -212,7 +212,7 @@ func SetupControllers(ctx context.Context, mgr ctrl.Manager, config *rest.Config
 		klog.Info("Setting up the memberCluster watcher for scheduler")
 		if err := (&membercluster.Reconciler{
 			Client:                    mgr.GetClient(),
-			SchedulerWorkQueue:        defaultSchedulingQueue,
+			SchedulerWorkqueue:        defaultSchedulingQueue,
```

Comment:
is Workqueue a word?

Reply:
Both spelling exist in the Kubernetes repo (and across the net); shall we have a preferred one? I guess `workqueue` is used because of the `workqueue` package 🤔

Reply:
prefer WorkQueue as they are two separate words.